vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/ZoteroDefaultMetaData.java

```diff
@@ -0,0 +1,118 @@
+package edu.asu.diging.vspace.referenceExpose;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+
+import edu.asu.diging.vspace.core.model.IReference;
+
+public class ZoteroDefaultMetaData implements IReferenceMetadata {
```

Comment:
It's a little confusing what all these classes do. This is all still pretty tightly coupled, I would change it like this:
- all instances of `IReferenceMetadata` should be services or components (maybe rename it to include the word formatter)
- ReferenceContext should be renamed to ReferencePresentationRegistry or something. It would get all beans of type IReferenceMetadata and put them in a map (identifier of formatter -> formatter instance)
- when we need a formatted metadata string, the identifier of the required formatter is used to retrieve the formatter via the registry

Reply:
presentation of metadata should not be part of any domain objects and it shouldn't be necessary to touch existing classes just to add another formatter.